Resources/Prototypes/_NF/PointsOfInterest/medical.yml

```
@@ -0,0 +1,50 @@
+# Author Info
+# GitHub: dustylens
+# Discord: dustylens
+
+# Maintainer Info
+# GitHub: ???
+# Discord: ???
+
+# Notes:
+# Medical ship garage, medical manager base and medical bounty staging area.
+- type: pointOfInterest
+  id: Medical
+  name: 'Medical Dispatch'
+  minimumDistance: 1650
+  maximumDistance: 3400
+  spawnGamePreset: [ NFAdventure, NFPirate ]
+  spawnGroup: Required
+  gridPath: /Maps/_NF/POI/medical.yml
+  addComponents:
+  - type: IFF
+    color: "#00b7eb"
+    readOnly: true
+  - type: Shuttle
+    angularDamping: 999999
+    linearDamping: 999999
+  - type: StationTransit
```

```suggestion
- type: pointOfInterest
  id: Medical
  parent: BasePOI
  name: 'Medical Dispatch'
  minimumDistance: 1650
  maximumDistance: 3400
  spawnGroup: Required
  gridPath: /Maps/_NF/POI/medical.yml
  addComponents:
  - type: IFF
    color: "#00b7eb"
  - type: StationTransit
```
A few of these fields can be inherited from the parent.
Not 100% sure these changes work, but should be roughly good.